guppylang/ast_util.py

```diff
@@ -335,12 +335,12 @@ def get_type(node: AstNode) -> "Type":
 def has_empty_body(func_ast: ast.FunctionDef) -> bool:
     """Returns `True` if the body of a function definition is empty.
 
-    This is the case if the body only contains a single `pass` statement or an ellipsis
+    This is the case if the body only contains a single `pass` statement or a constant
     `...` expression.
     """
     if len(func_ast.body) == 0:
         return True
     if len(func_ast.body) > 1:
         return False
     [n] = func_ast.body
-    return isinstance(n, ast.Expr) and isinstance(n.value, ast.Ellipsis)
+    return isinstance(n, ast.Expr) and isinstance(n.value, ast.Constant)
```

Comment:
```suggestion
    return isinstance(n, ast.Expr) and isinstance(n.value, ast.Constant) and n.value.value is Ellipsis
```
I am not certain about this, but it looks to me from https://docs.python.org/3/library/ast.html#ast.Constant that it should be thus.